### DISCUSSION TITLE
dotnet restore - Since when are revocation checks enabled for Linux?

### DISCUSSION THREAD

**Discussion by stefan-hartmann-lgs** [NuGet/Home#12028] [General]
In the past revocation checks have been disabled under linux for `dotnet restore`. It seems like that since `6.0.400` this seems not to be the case anymore and they are enabled. When did this behaviour change happen - could not find anything in the release notes!? Thanks

**Comment by PathogenDavid** [comment]
I would assume this is part of [signed package verification on Linux being re-enabled](https://devblogs.microsoft.com/nuget/announcing-nuget-6-3-transitive-dependencies-floating-versions-and-re-enabling-signed-package-verification/#re-enable-signed-package-verification-on-linux-by-default). (See also https://github.com/dotnet/core/issues/7688)

**Reply by stefan-hartmann-lgs** [reply]
Thanks @PathogenDavid - this is exactly what I was looking for - cheers
